src/main/java/org/rciam/plugins/groups/representations/GroupsPager.java

```diff
@@ -5,15 +5,15 @@
 import org.keycloak.representations.idm.GroupRepresentation;
 
 public class GroupsPager {
-    private List<GroupRepresentation> results;
+    private List<? extends GroupRepresentation> results;
```

Comment:
Is any need to support GroupRepresentation extension?
I propose to remove all `? extends `.